### DISCUSSION TITLE
Lock file not found

### DISCUSSION THREAD

**Discussion by greatbyte** [community/community#130399] [Other Feature Feedback, Questions, & Ideas]
I wanted to commit and discovered an error message:

> A lock file already exists in the repository, which blocks this operation from completing.

The next thing I did was to google the answer. However, I unchecked the hidden files and folder to have access to the locked file that is causing the problem. Unfortunately, I could not manually delete the index.lock file. See the pic.
![gIT1](https://github.com/community/community/assets/13262746/9358f422-caa4-4036-92d3-64ee37b34aeb)
Please help!

**Comment by ghostinhershell** [comment]
Thanks for posting in the GitHub Community, @greatbyte!

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the New to GitHub category is solely related to conversations around getting started on using GitHub and the community for beginners of GitHub, Community on Discussions, and those new to different areas of the product alike! This question should be in the General category. I've gone ahead and moved it for you. Good luck!
